java/src/com/ibm/streamsx/rest/Operator.java

```diff
@@ -50,13 +51,27 @@
     @Expose
     private String self;
 
-    /**
-     * this function is not intended for external consumption
-     */
-    void setConnection(final StreamsConnection sc) {
+    private void setConnection(final StreamsConnection sc) {
         connection = sc;
     }
 
+    static final List<Operator> getOperatorList(StreamsConnection sc, String operatorsList) {
+        List<Operator> opList;
+        OperatorArray opArray;
+        try {
+            opArray = new GsonBuilder().excludeFieldsWithoutExposeAnnotation().create().fromJson(operatorsList,
+                    OperatorArray.class);
+
+            opList = opArray.operators;
+            for (Operator op : opList) {
+                op.setConnection(sc);
+            }
+        } catch (IllegalStateException e) {
+            opList = new ArrayList<Operator>();
```

Comment:
this is a runtime exception that can't really be acted upon by the user.  so by returning the empty list, most code will just iterate through, versus fail.  if it is a persistent problem, (i.e, the structure changed enough that Gson.fromJson() isn't working, there will be other failures that will trigger.   

i'll make the change to Collections.emptyList() as the return for all of these, instead of a new ArrayList.